pkg/process/checks/templates/processes.tmpl

```diff
@@ -0,0 +1,35 @@
+Processes
+=========
+{{- range $process := .Processes}}
+> PID: {{ .Pid }} NSPID: {{ .NsPid }} PPID: {{ .Command.Ppid }}
+  Container ID: {{ .ContainerId }}
+  Exe: {{ .Command.Exe }}
+  Args:{{ range .Command.Args}} '{{.}}'{{ end }}
+  Cwd: {{ .Command.Cwd }}
+{{- with .User }}
+  User: {{.Name}} Uid: {{.Uid}} Gid: {{.Gid}} Euid: {{.Euid}} Egid: {{.Egid}} Suid: {{.Suid}} Sgid: {{.Sgid}}
+{{- end }}
+  Create Time: {{ .CreateTime }}
+{{- with .Memory }}
+  Memory:
+    Rss:    {{.Rss}}
+    Vms:    {{.Vms}}
+    Swap:   {{.Swap}}
+    Shared: {{.Shared}}
+    Text:   {{.Text}}
+    Lib:    {{.Lib}}
+    Data:   {{.Data}}
+    Dirty:  {{.Dirty}}
+{{- end }}
+{{- with .Cpu }}
+  CPU: Total: {{ .TotalPct }}% System: {{ .SystemPct }}% User: {{ .UserPct }}%
+  Threads: {{ .NumThreads }}
+  Nice: {{ .Nice }}
+{{- end }}
+  Open Files: {{ .OpenFdCount }}
```

Comment:
We should probably also add `Voluntary and Involutary Context Switches`:
https://github.com/DataDog/datadog-agent/pull/11487/files#diff-4d8b6b8ff0d5bd68fe74f4943931e3ef84d5160b574be81d33f9694fe55b755eR24-R25

Reply:
Added thank you!